swan-lake/development-tutorials/test-document-the-code/test-ballerina-code/execute-tests-in-parallel.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: ballerina-testing-code-left-nav-pages-swanlake
+title: Execute tests in parallel
+description: Learn how to execute tests in parallel.
+keywords: ballerina, programming language, testing, test execution
+permalink: /learn/test-ballerina-code/execute-tests-in-parallel/
+active: execute-tests-in-parallel
+intro: Ballerina test framework allows developers to execute test cases in parallel, which can significantly reduce the overall test execution time, especially for large codebases with a large number of test cases.
+---
+
+## Enable parallel execution of tests
+
+Tests are executed serially by default. The user has to explicitly activate the parallel execution by providing the `--parallel` flag as follows.
+
+```bash
```

Comment:
Remove bash. If we put bash, the code will be evaluated as a bash script and will result in incorrect syntax highlighting.
```suggestion
```
```